.github/workflows/linux.yml

```diff
@@ -15,55 +15,45 @@ permissions:
   security-events: write
 
 jobs:
-  cleanup:
-    runs-on: [ self-hosted, linux-do ]
-    container: grafin1992/nemu-ci:v1.2
-    steps:
-      - name: Clean workspace
-        run: rm -rf nemu
-
   clone:
     runs-on: [ self-hosted, linux-do ]
-    container: grafin1992/nemu-ci:v1.2
-    needs: cleanup
+    container: voidso/nemu-ci:v1.0
 
     steps:
       - name: Get nEMU sources
-        run: git clone --depth 1 https://github.com/nemuTUI/nemu
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
 
   configure:
     runs-on: [ self-hosted, linux-do ]
-    container: grafin1992/nemu-ci:v1.2
+    container: voidso/nemu-ci:v1.0
     needs: clone
 
     steps:
       - name: Configure nEMU build
-        working-directory: ./nemu
         run: |
-          mkdir build && cd build
```

Comment:
Decided to build inplace? This also tested, that out-of-tree build works fine, maybe leave it as is?

Reply:
Added step